model/src/main/proto/topic.proto

```diff
@@ -257,6 +257,58 @@ message TopicProgress {
 
   // Map from story ID to StoryProgress.
   map<string, StoryProgress> story_progress = 2;
+
+  // Timestamp to record last time the exploration was played in ms.
+  int64 last_played_timestamp = 3;
+}
+
+// A structure corresponding to the Promoted Stories. This structure is set up
+// to properly account for recently played stories, recommended stories
+// and coming soon topics.
+message RecommendedActivityList {
+  oneof recommendation_type {
+    RecommendedStoryList recommended_story_list = 1;
+    ComingSoonTopicList coming_soon_topic_list = 2;
+  }
+}
+
+// Corresponds to the list of stories the player is currently playing across all topics and Recommended stories.
```

Comment:
There is some inconsistency in the way you are mentioning `recommended`, `promoted` and `stories`.

For example in `Line 65` you are writing `Stories` instead of `stories` and in this line you are writing it as `stories`. 
Try to make this consistent across all comments. 
I suggest using `promoted stories`, `stories`, `recommended stories`, meaning everything in small caps.

Reply:
Done.